cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -1797,6 +1797,15 @@ impl_runtime_apis! {
 			genesis_config_presets::preset_names()
 		}
 	}
+
+	impl xcm_runtime_apis::trusted_query::TrustedQueryApi<Block> for Runtime {
+		fn is_trusted_reserve(asset: VersionedAsset, location: VersionedLocation) -> Result<bool, xcm_runtime_apis::trusted_query::Error> {
```

Comment:
You can define a
```rust
type XcmTrustedQueryResult = Result<bool, xcm_runtime_apis::trusted_query::Error>;
```
at the top level of files like this on so then the runtime apis have shorter/more-readable signature:
```suggestion
		fn is_trusted_reserve(asset: VersionedAsset, location: VersionedLocation) -> XcmTrustedQueryResult {
```